src/editor/gltf/GLTFExporter.js

```diff
@@ -1261,6 +1261,47 @@ class GLTFExporter {
     return skinIndex;
   }
 
+  /**
+   * Process camera
+   * @param  {THREE.Camera} camera Camera to process
+   * @return {Integer}      Index of the processed mesh in the "camera" array
+   */
+  processCamera(camera) {
+    if (!this.outputJSON.cameras) {
+      this.outputJSON.cameras = [];
+    }
+
+    const isOrtho = camera.isOrthographicCamera;
+
+    const gltfCamera = {
+      type: isOrtho ? "orthographic" : "perspective"
+    };
+
+    if (isOrtho) {
+      gltfCamera.orthographic = {
+        xmag: camera.right * 2,
+        ymag: camera.top * 2,
+        zfar: camera.far <= 0 ? 0.001 : camera.far,
+        znear: camera.near < 0 ? 0 : camera.near
+      };
+    } else {
+      gltfCamera.perspective = {
+        aspectRatio: camera.aspect,
+        yfov: _Math.degToRad(camera.fov),
+        zfar: camera.far <= 0 ? 0.001 : camera.far,
+        znear: camera.near < 0 ? 0 : camera.near
+      };
+    }
+
+    if (camera.name !== "") {
+      gltfCamera.name = camera.type;
```

Comment:
Is this intentional that you save type as name if name is not empty?

Reply:
This code was copied from the upstream GLTFExporter at the time. It was removed earlier in a Spoke commit so I just brought it back.

Reply:
I think this should be fixed in the official `GLTFExporter` side.